code/modules/research/designs/weapons.dm

```diff
@@ -552,12 +585,94 @@
 	build_path = /obj/item/ammo_storage/magazine/a12mm/empty
 
 //Misc
-/datum/design/vectorreceiver
```

Comment:
Was there a reason to remove this?

Reply:
It was unused, so I didn't see a reason to keep it around. Vector receivers seem like an unimplemented feature: all freshly printed Vectors come with one, and the "unrestricted" receiver doesn't seem to be available anywhere and causes runtimes when you try to load it.

Reply:
Do you have any issues with that bug in it? Id like to see them since that was my project in the past. As for why it was intended to be a cheaper way to get just the receiver if you lost the other or so you could have multiple on hand for different mag types. I also urge you not to remove the DESIGN because it allows mechanics to make more with consistent prices. The lathe can go if you feel strongly about it but the design should stay.